packages/types/src/strategies.ts

```diff
@@ -0,0 +1,12 @@
+import { OAuthProvider } from './oauth';
+import { Web3Provider } from './web3';
+
+export type PasswordStrategy = 'password';
+export type PhoneCodeStrategy = 'phone_code';
+export type EmailCodeStrategy = 'email_code';
+export type EmailLinkStrategy = 'email_link';
+export type TicketStrategy = 'ticket';
+export type OrganizationTicketStrategy = 'ticket';
+
+export type OAuthStrategy = `oauth_${OAuthProvider}`;
+export type Web3Strategy = `web3_${Web3Provider}_signature`;
```

Comment:
:upside_down_face: Perhaps it's better if we add the provider to the type name here as well? `Web3MetamaskStrategy` instead of simply `Web3Strategy`. What happens if we get another web3 provider in the future?